addons/lunch/data/mail_template_data.xml

```diff
@@ -43,13 +44,28 @@
         </p><p>
         Here is, today orders for ${order['company_name']}:
         </p>
+        <table>
+            <thead>
+                <tr>
+                    <th style="width: 100%; min-width: 96px; font-size: 13px;" align="left"><strong>Location</strong></th>
+                </tr>
+            </thead>
+            <tbody>
+                % for site in sites
```

Comment:
This is not a valid html construct. You cannot have something between a tbody and its tr? This means your jinja is likely to be moved somewhere else in your html code depending on your browser when you edit it and broken afterwards. It is probably better to find a lightweight display, don't think a table is that necessary.